develop/plone/searching_and_indexing/external-catalogs.rst

```diff
@@ -0,0 +1,64 @@
+=======================
+Using external catalogs
+=======================
+
+.. admonition:: Description
+
+        The Plone catalog can be extend to use external catalogs like Solr or elastic search. Add-ons like collective.solr use that to hook into the catalog API and do some Indexing outside of Plone in Solr, which increases performance and flexibility of indexing a lot.
+
+.. contents :: :local:
+
+
+Implement IIndexingQueueProcessor
+---------------------------------
+
+To hook into the catalog one can implement the IIndexingQueueProcessor `interface from Products.CMFCore <https://github.com/plone/Products.CMFCore/blob/master/Products/CMFCore/interfaces/_tools.py>`_.
+
+
+.. note:: For Plone < 5.1 you need to use the `interfaces from collective.indexing <https://github.com/plone/collective.indexing/blob/master/src/collective/indexing/interfaces.py>`_ package!
+
```

Comment:
I do not get this :) , we do not have PLone 5.1 yet, so this is confusing, if you want to add this would it be possible to rewrite this a bit and please do not use _..note::_ here, for this you should use _.. versionadded:: version_ more info here: http://www.sphinx-doc.org/en/stable/markup/para.html and a example:

.. versionadded:: 2.5
   The *spam* parameter.